README.md

```diff
@@ -5,7 +5,7 @@ This is a skeleton repository that can be used for testing or as a quick way for
 If you are new to the Nubisproject you will need to set up some [prerequisites](https://github.com/Nubisproject/nubis-docs/blob/master/PREREQUISITES.md). 
 
 ## Get the code
-Next grab the latest [release](https://github.com/Nubisproject/nubis-skel/releases), extract it and copy the *nubis* directory into your code base.
+Next grab the latest [release](https://github.com/Nubisproject/nubis-skel/releases), extract it and copy the *nubis* directory into your code base. You also need to add *nubis/cloudformation/parameters.json* to your *.gitignore* file. See the project [.gitignore](.gitignore) file for an example.
```

Comment:
It is included in the release notes. My assumption here is that people are copying the nubis folder from the release artifact into their preexisting project. I therefore assume that they already have their own .gitignore file and simply need to add this bit. If they do not have one then they could certainly use the one that comes with the bundle.